src/app/[locale]/(pages)/profileAccessBlocked/page.tsx

```diff
@@ -6,16 +6,16 @@ import { ButtonNaked } from '@pagopa/mui-italia';
 import { useTranslations } from 'next-intl';
 import { Introduction } from '../../_component/introduction/introduction';
 import Firework from '../../_icons/firework';
-import { commonBackground } from '../../_utils/styles';
 import { ROUTES } from '../../_utils/routes';
+import { commonBackground } from '../../_utils/styles';
 
 const Profile = () => {
   const t = useTranslations('ioesco');
   const bgColor = 'background.paper';
   return (
     <Grid sx={commonBackground}>
       <Introduction
-        title={t('common.hello', { Nome: 'Mario' })}
+        title={t('common.hello', { nome: 'Mario' })}
```

Comment:
why you don't use the name from the token? (i have notice this issue in some pages)

Reply:
The name variable will not be read from the session storage but will be returned from a api call that will be integrated in the future. For the moment we will leave it in the code until the new API is created and integrated.